backend/src/main/resources/application.yaml

```diff
@@ -48,6 +48,8 @@ simple-report:
     max-csv-rows: 999
     api-key: ${DATAHUB_API_KEY:MISSING}
     secret-slack-notify-webhook-url: ${SECRET_SLACK_NOTIFY_WEBHOOK_URL:MISSING}
+    upload-schedule: "0 0 11 * * *" # Daily at 11:00 AM Eastern Time
```

Comment:
Personally I would update to run the job every 15 minutes in prod too. Would be good to fast follow this PR with the update

Reply:
Every 2 hours was agreed upon, so I added that to `application-azure-prod.yaml`.